posthog/hogql/ast.py

```diff
@@ -40,6 +41,21 @@ class VariableDeclaration(Declaration):
     expr: Optional[Expr] = None
 
 
+@dataclass(kw_only=True)
+class Block(Expr):
```

Comment:
Having a util that creates a `Block` from similar interface as a `ast.Placeholder` could be nice:

```python
def block_from_field_chain(chain: list[str | int]) -> ast.Block:
	return ast.Block(....)
```